frontend/src/components/Button/BannerButton.tsx

```diff
@@ -0,0 +1,68 @@
+import React from 'react'
+import { Link } from 'react-router-dom'
+import './BannerButton.css'
+import { useMediaQuery, useTheme } from '@mui/material';
+
+const ResponsiveComponent = () => {
```

Comment:
Based from the code this is only used for the banner right? Of so, make the naming more specific.